README.md

```diff
@@ -59,12 +59,22 @@ pip3 install .
 
 ### PDF Files
 
+#### Local Compilation
+
 To compile PDF files, the `pdflatex` program is used.
 `pdflatex` comes installed with most standard [LaTeX](https://en.wikipedia.org/wiki/LaTeX) packages.
 
 By default, your [PATH](https://en.wikipedia.org/wiki/PATH_(variable)) will be searched for `pdflatex`.
 To specify the path to your `pdflatex` binary, you can use the `--pdflatex-bin-path` flag.
 
+#### Docker Compilation
+
+PDF File compilation using Docker Command.
```

Comment:
This needs more description.
For example:
 - What image do we use?
 - What are the requirement?
 - Are there any caveats?
 - Does it require any flags or options?